README.rst

```diff
@@ -30,27 +29,24 @@ from ``benchopt`` Command Line Interface (CLI).
 
 To install ``benchopt``, start by creating a new ``conda`` environment and then activate it
 
-.. code-block::
+.. code-block:: bash
 
-    conda create -n benchopt python
+    conda create -n benchopt
```

Comment:
Why remove this? when using `conda-forge` as default, I think the new env don't have python so it is better to leave it
```suggestion
    conda create -n benchopt python
```